source/jsonnet/northern-ireland/individual/blocks/personal-details/date_of_birth.jsonnet

```diff
@@ -4,13 +4,9 @@ local rules = import 'rules.libsonnet';
 local question(title, mandatory) = {
   id: 'date-of-birth-question',
   title: title,
-  guidance: {
-    contents: [
-      {
-        description: 'For example, 31 3 1970',
-      },
-    ],
-  },
+  description: [
+    'For example, 31 3 1970',
```

Comment:
This date doesn't appear to match in the spreadsheet - year should be 1980

Reply:
As discussed, didn't change cause the sheet was wrong. But checked with Donna and have updated to make it consistent.